pkg/cnservice/server.go

```diff
@@ -174,6 +174,10 @@ func (s *service) createMOServer(inputCtx context.Context, pu *config.ParameterU
 		metric.InitMetric(moServerCtx, ieFactory, pu, 0, metric.ALL_IN_ONE_MODE)
 	}
 	frontend.InitServerVersion(pu.SV.MoVersion)
+	err := frontend.InitSysTenant(moServerCtx)
```

Comment:
This should not be initialized in cn

Reply:
Does not really matter -- as long as it init only once per SYSTEM, not per restart or create new service.

This Init runs sql inside, so need CN.

Reply:
In fact, I promise you that there will be upgrade, so really, you need to start a CN and runs upgrade there -- Let's hope there will never be downgrade :)

Reply:
My concern is how this new dn will be guaranteed to be initialised correctly after the dn is expanded.

Reply:
Well, for the time being I'm not quite sure how to do this logic better, but 0.6 should definitely work this way (although personally it feels a bit odd). I even think it's better to handle it in ha than cn.

Reply:
I have confirmed it with @reusee who said it was ok here.